src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -193,8 +193,11 @@ private void PreloadLibraries(IEnumerable<string> preloadLibraries)
         {
             importedLibraries.AddRange(preloadLibraries);
 
-            foreach (var library in importedLibraries)
-                CompilerUtils.TryLoadAssemblyIntoCore(LibraryManagementCore, library);
+            var importNodes = preloadLibraries.Select(lib => new ImportNode { ModuleName = lib });
+            var codeGen = new CodeGenDS(importNodes);
+            string code = codeGen.GenerateCode();
+
+            CompilerUtils.TryLoadAssembliesIntoCore(LibraryManagementCore, code);
```

Comment:
Unrelated to this task but this function loads all path manager libraries together instead of one-by-one, which was what was happening earlier. This is now consistent with the loading of libraries in LiveRunner core.